collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,346 @@
+
+GLTF is the main 3D asset format supported by Godot, and it even acts as a backbone for FBX and `.blend` file imports. Because of that, the import team is always dedicating a lot of attention to improving our support of GLTF imports, various extensions, and fixing related bugs. One of the new supported GLTF extensions is `KHR_materials_emissive_strength`, which is responsible for representing Blender's emission multiplier, among other things. [Gordon MacPherson](https://github.com/RevoluPowered) implemented it for imported assets, and [wojtekpil](https://github.com/wojtekpil) then ensured it was also supported when exporting GLTF assets from Godot ([GH-78621](https://github.com/godotengine/godot/pull/78621), [GH-79421](https://github.com/godotengine/godot/pull/79421)).
+
+Exporting meshes and models from Godot is an important part of some workflows, such as blocking out level geometry in the editor and then enhancing it in modeling software. Which is why [Aaron Franke](https://github.com/aaronfranke) spent time addressing several issues and inconsistencies in the process, as well as fixing other asset pipeline bugs ([GH-79533](https://github.com/godotengine/godot/pull/79533), [GH-79623](https://github.com/godotengine/godot/pull/79623), [GH-79636](https://github.com/godotengine/godot/pull/79636), [GH-79801](https://github.com/godotengine/godot/pull/79801)).
+
+One more notable change to the import process is the newly added KTX image format support ([GH-76572](https://github.com/godotengine/godot/pull/76572)). This type of image is often used for Basis Universal in GLTF. That was a pretty impressive first contribution for [acazuc](https://github.com/acazuc)!
+
+
+### Input
+
+A lot of work is happening behind the scene around Viewport mouse handling. [Markus Sauermann](https://github.com/Sauermann), project's dedicated input maintainer, spent a lot of time making sure mouse and focus event behave predictably across multiple viewports and windows ([GH-67791](https://github.com/godotengine/godot/pull/67791) and many follow-ups). This is an area that is very difficult to get right, as every platform and environment has its own gimmicks and compromises. Make sure to report any issues related to mouse behavior that you find!
+
+Godot 4.2 also attempts to resolve two major issues related to gamepads. First of all, contributor [Eoin O'Neill](https://github.com/Eoin-ONeill-Yokai) offers a fix to the Steam Input issue which caused some gamepad events to be handled twice ([GH-76045](https://github.com/godotengine/godot/pull/76045)). Second issue relates to an infamous bug affecting actions with multiple input methods assigned to them, such as character movement controlled by WASD and a thumbstick/D-Pad ([GH-80859](https://github.com/godotengine/godot/pull/80859), [GH-81170](https://github.com/godotengine/godot/pull/81170) by [Tomasz Chabora](https://github.com/KoBeWi)).
```

Comment:
```suggestion
Godot 4.2 also attempts to resolve two major issues related to gamepads. First of all, contributor [Eoin O'Neill](https://github.com/Eoin-ONeill-Yokai) offers a fix to the Steam Input issue which caused some gamepad events to be handled twice ([GH-76045](https://github.com/godotengine/godot/pull/76045)). The second issue relates to an infamous bug affecting actions with multiple input methods assigned to them, such as character movement controlled by WASD and a thumbstick/D-Pad ([GH-80859](https://github.com/godotengine/godot/pull/80859), [GH-81170](https://github.com/godotengine/godot/pull/81170) and was resolved by [Tomasz Chabora](https://github.com/KoBeWi)).
```

Reply:
Here the credit is not a part of the sentence, it's just a reference to the PR author in the parentheses. I edited your comment so I can apply the rest of the suggestion!